action.yml

```
@@ -56,6 +56,10 @@ inputs:
    description: "Relative path under GITHUB_WORKSPACE to the repository"
    required: false
    default: '.'
+  skip_initial_fetch:
+    description: "Skip fetching more histories when the repo is shallow, this is passed to [Changed Files](https://github.com/tj-actions/changed-files)" 
```

I'll recommend describing the input itself as opposed to the internal workings 

```suggestion
    description: |
      Skip initially fetching additional history to improve performance for shallow repositories.
      NOTE: This could lead to errors with missing history. It's intended to be used when you've fetched all necessary history to perform the diff.
```